process/phases.md

```diff
@@ -0,0 +1,150 @@
+# WebAssembly W3C Process
+
+This file describes how feature proposals will progress through
+the standardization process.
+
+In general, the process moves forward through a series of numbered phases.
+However, if issues are uncovered or consensus devolves,
+proposals should back up to the appropriate prior step.
+ 
+## 0. Feature Proposal [CG]
+
+Entry requirements:
+  * A CG member has an idea.
+ 
+During this phase a new repository forking the spec repo is created (CG members
+can request this). PRs + Issues iterate on the design of the feature. If human
+readable design repo style .md files are useful, these are added to describe
+the feature as it's being designed. A tracking issue is created in the design
+repo.
+ 
+If relevant to demonstrate the viability of a feature, prototype
+implementations of the feature are implemented in interested engines (possibly
+on a branch).
```

Comment:
So engine struck me as more clear, as embedder sounds relevant if the feature can wholely be done outside the core engine, but odd in reference to some part of the core format?
See below on the other variation on this.